masterlist.yaml

```diff
@@ -17651,6 +17656,12 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("CFTO - Immersive Citizens Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Creation Club Bittercup'
+          - '[CFTO Bittercup Compatibility Patch](https://www.nexusmods.com/skyrimspecialedition/mods/60587/)' 
+        conditions: 'active("CFTO.esp") and active("cckrtsse001_altar.esl") and not active("CFTO_Bittercup_Patch.esp")'
+        
```

Comment:
This line can also be removed, we don't add empty lines inbetween different data nodes of the same plugin. So it is
```yaml
  - name:
    url:
    after:
    msg:
    clean:
```
rather than
```yaml
  - name:
    url:
    after:
    msg:

    clean:
```

Reply:
Taken care of. I actually hadn't even noticed I'd added it.